File: server/Education.php
<?php
	$con=mysqli_connect("localhost","root","qwerty","db1");

    $user_id=$_POST["tc_no"];
	$okul_adi=$_POST["okul_adi"];
	$bolum=$_POST["bolum"];
	$mezuniyet_yili=$_POST["mezuniyet_yili"];

	$statement=mysqli_prepare($con,"INSERT INTO education (tc_no,okul_adi,dolum,mezuniyet_yili) VALUES (?,?,?,?)");
	mysqli_stmt_bind_param($statement,"issi",$tc_no,$okul_adi,$bolum,$mezuniyet_yili);
	mysqli_stmt_execute($statement);

	mysqli_stmt_close($statement);

	mysqli_close($con);
?>

File: app/src/main/java/ru/mail/loginregister/siniflar/Education.java
package ru.mail.loginregister.siniflar;

public class Education {

    private String okul_adi,bolum;
    private int mezuniyet_yili;

    public Education(String okul_adi,String bolum,int mezuniyet_yili){
        this.okul_adi=okul_adi;
        this.bolum=bolum;
        this.mezuniyet_yili=mezuniyet_yili;
    }

    public String getOkul_adi(){return okul_adi;}

    public String getBolum(){return bolum;}

    public int getMezuniyet_yili(){return mezuniyet_yili;}

}
